makefiles/Makefile.devhub-content

```diff
@@ -46,6 +46,7 @@ next-gen-html: update-snooty
 
 next-gen-html-publish: update-snooty
 	# snooty parse and then build-front-end
+	curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/devhub-content.yaml > ${REPO_DIR}/published-branches.yaml
```

Comment:
While we do use published_branches for entitlements checks, we don't use them for actual publishing for devhub-content, so we can remove this line for this repo only.